air/src/execution_step/errors/catchable_errors.rs

```diff
@@ -32,7 +32,7 @@ use std::rc::Rc;
 /// Catchable errors arisen during AIR script execution. Catchable here means that these errors
 /// could be handled by a xor instruction and their error_code could be used in a match
 /// instruction.
-#[derive(ThisError, EnumDiscriminants, Debug)]
+#[derive(ThisError, EnumDiscriminants, Debug, Clone)]
```

Comment:
Can we keep this value in a `Rc` instead?

Reply:
There is ExecutionError instance in Xor::execute that must be cloned before send down anyway.